### PR TITLE
Content and big fixes for household characteristics

### PR DESCRIPTION
**Pull request by paulrobertlloyd**
* Ensures options and answer labels use the correct content
* Only show one row for tenant/person’s age
* Bug fixes (spotted while editing JSON)
* Update tests

NOTE: This PR does not address the issue around deriving the correct number of people in the household, or adjust the answer label that relates to this.

### CODE REVIEW

File: spec/fixtures/forms/2021_2022.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
               "person_1_age": {
```
Comment (kosiakkatrina) [inline spec/fixtures/forms/2021_2022.json:31]: We don't care too much about the wording in our test fixtures. The idea behind it being, if we change content for the actual form that doesn't impact the tests. It doesn't hurt having this as a change, but for the next time, it's not necessary

Comment (paulrobertlloyd) [reply spec/fixtures/forms/2021_2022.json:31]: Yeah, I’m always unsure how important/useful it is to keep the fixtures and tests up to date with the actual form. OCD usually gets the better of me! 🤪

**Review by kosiakkatrina — APPROVED** [review]
Looks good. It seems like the mappings for ecstat(1-8) still need to be updated, but I assume it's not part of this PR

**Comment by paulrobertlloyd** [comment]
> Looks good. It seems like the mappings for ecstat(1-8) still need to be updated, but I assume it's not part of this PR

Thanks @kosiakkatrina; not sure what needs to be updated changed with regards to this issue, so left values as is for now.

**Review by dushan-madetech — APPROVED** [review]
Looks good to me 👍. Happy for you to 🚢 it when you're ready for it to go into main

**Pull request merged**